recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
 {% set name = "pysteps" %}
-{% set version = "1.5.1" %}
+{% set version = "1.6.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 0bbe82757919fd942e8d6f1138190851e02f4f417290c19517f12ee144405b23
+  sha256: 3a6ba22e5510d8393ed2f18aefe131daef39f85bc6737e6c9ff2af43cddb135e
 
 build:
-  number: 1
+  number: 0
   skip: true  # [win and py>38]
```

Comment:
```suggestion
```
I suppose we can now remove this selector for sckipping win builds for py>38, or at least try. @aperezhortal ?

Reply:
Good idea. I imagine that by now, the builds for the required dependencies are in conda forge. It worth giving it a try.